client/app/components/dynamic-form/DynamicForm.jsx

```diff
@@ -77,10 +77,12 @@ class DynamicForm extends React.Component {
   handleSubmit = (e) => {
     this.setState({ isSubmitting: true });
     e.preventDefault();
+    const notEmpty = (value, key) => !(this.props.fields.find(f => f.name === key).initialValue === undefined && value === ''); // rejects empty values, unless they are intentionally set to empty
```

Comment:
So if some field had a value before, but now I clear it out, we will still send an empty value?

Reply:
Yes, an empty string will be sent.

Reply:
This is a problem, because some values will work with a null but not an empty string. In most cases I guess we handled it (like the Presto data source), but not sure if in all of them.

Maybe set it to null in such case?

Another option, is to remove these values on the backend when merging with old params. Or maybe not merge at all?

Reply:
> Maybe set it to null in such case?
Seems to work well.